doc_files/iter8_install.md

```diff
@@ -1,52 +1,46 @@
-```
+In case you need to customize the installation of iter8's latest release, use the Helm charts listed below:
 
-### Customized installation
+* _iter8-analytics_: [https://github.com/iter8-tools/iter8-analytics/releases/latest/download/iter8-analytics-helm-chart.tar](https://github.com/iter8-tools/iter8-analytics/releases/latest/download/iter8-analytics-helm-chart.tar)
 
-In case you need to change the default configuration options used in the quick installation, use the helm charts directly by cloning the projects:
+* _iter8-controller_: [https://github.com/iter8-tools/iter8-controller/releases/latest/download/iter8-controller-helm-chart.tar](https://github.com/iter8-tools/iter8-controller/releases/latest/download/iter8-controller-helm-chart.tar)
+
+### Installing an older release
+
+In case you need to install an old iter8 release, please, refer to its corresponding documentation in the link below. In the URL below, you need to replace the string `<release>` with the string corresponding to your desired release. For example, a valid release string would be `v0.0.1`.
 
-```bash
-git clone git@github.com:iter8-tools/iter8-analytics.git
-git clone git@github.com:iter8-tools/iter8-controller.git
+```
+https://github.com/iter8-tools/docs/tree/<release>
 ```
 
-The _iter8-analytics_ helm chart is [here](https://github.com/iter8-tools/iter8-analytics/tree/master/install/kubernetes/helm/iter8-analytics), and the _iter8-controller_ helm chart is [here](https://github.com/iter8-tools/iter8-controller/tree/master/install/helm/iter8-controller).
+Note that the URL above points to the GitHub branch and tag corresponding to your desired release in the documentation repository.
 
-**Note on Prometheus:** In order to make assessments on canary releases, _iter8_analytics_ needs to query metrics collected by Istio and stored on Prometheus. The default values for the helm chart parameters (used in the quick installation) point _iter8_analytics_ to Prometheus at `http://prometheus.istio-system:9090`, which is the default internal Kubernetes URL of Prometheus installed as an Istio addon. If your Istio installation is shipping metrics to a different Prometheus installation, you need to set the _iter8-analytics_ helm chart parameter `iter8Config.metricsBackendURL` to your Prometheus `host:port`.
+**Note on Prometheus:** In order to make assessments, _iter8_analytics_ needs to query metrics collected by Istio and stored on Prometheus. The default values for the helm chart parameters (used in the quick installation) point _iter8_analytics_ to Prometheus at `http://prometheus.istio-system:9090`, which is the default internal Kubernetes URL of Prometheus installed as an Istio addon. If your Istio installation is shipping metrics to a different Prometheus installation, you need to set the _iter8-analytics_ Helm chart parameter `iter8Config.metricsBackendURL` to your Prometheus `host:port`.
```

Comment:
I seem to have changed the parameter in my PR. It should be `metricsBackend.url`
Also, it needs to be a full url including protocol (http or https). Does `host:port` fully capture that?

Reply:
I see. The parameter we currently have in the master branch looks like this:

```
metricsBackendURL: "http://prometheus.istio-system:9090"
```
Note that it already have the protocol in the URL. So, I do not understand your comment about the lack of protocol.

So, I do not know what the best approach here is. We could make the text reflect exactly the  parameter structure yet to be merged, but that will be inconsistent with the Helm chart for 0.0.1. Alternatively, I can make the text more generic and tell the users to look for the parameter without telling then exactly what the parameter name(s) is(are).

Reply:
Okay, I decided to make the Prometheus-related instructions more generic, and also mentioned the authentication business briefly.

Reply:
clever solution. works for me.